build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     ext.kotlin_ktx_version = '1.2.0'
     ext.wordPressUtilsVersion = '1.40.0'
     ext.detektVersion = '1.15.0'
-    ext.gutenbergMobileVersion = 'v1.52.0'
+    ext.gutenbergMobileVersion = 'develop-b34dd4a03c462e285aab5a391ebf84799fbba48b'
```

Comment:
@jd-alexander I'd like to confirm with you that this is the proper way to set the Gutenberg Mobile ref before merging this PR.

Reply:
@fluiddot yes it. Once you have verified that the editor binary has been built on S3 you are good to go.

Reply:
I confirmed that the editor binary was uploaded, in fact, all checks are 🟢  in the [commit](https://github.com/wordpress-mobile/gutenberg-mobile/commit/b34dd4a03c462e285aab5a391ebf84799fbba48b).

Ok, thank @jd-alexander for confirming it, I'll merge it 🎊 !